CHANGELOG.md

```diff
@@ -5,27 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
```

Comment:
these 2 rows/ info are useless now. suggest remove

Reply:
Will remove in the next release.